### PR TITLE
CI Update - .NET Core 3.0 Preview 4

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: xml/Microsoft.Win32.SafeHandles/SafeMemoryMappedFileHandle.xml [hunk 1]
```diff
@@ -98,7 +98,7 @@
         <AssemblyName>System.Core</AssemblyName>
       </AssemblyInfo>
       <Attributes>
-        <Attribute FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0">
+        <Attribute FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2">
```
[the remark this answers is not in the document]
Comment (mairaw) [reply xml/Microsoft.Win32.SafeHandles/SafeMemoryMappedFileHandle.xml]: It does indeed. @joelmartinez can you check if this is expected? If not, I can open a bug.

File: xml/System.Buffers/SequenceReaderExtensions.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
       </ReturnValue>
       <Parameters>
         <Parameter Name="reader" Type="System.Buffers.SequenceReader&lt;System.Byte&gt;" RefType="this" />
-        <Parameter Name="value" Type="System.Int64" RefType="out" />
+        <Parameter Name="value" Type="System.Int16" RefType="out" />
```
[the remark this answers is not in the document]
Comment (mairaw) [reply xml/System.Buffers/SequenceReaderExtensions.xml]: This is a "known" issue but I'm not sure if we have a bug tracking these APIs. @joelmartinez can you confirm if you have a tracking bug for this? I couldn't find it on https://github.com/mono/api-doc-tools

[the remark this answers is not in the document]
Comment (joelmartinez) [reply xml/System.Buffers/SequenceReaderExtensions.xml]: @mairaw no I'm not sure I had seen this issue before … hmm, so the issue is that there's three overloads in the actual assemblies, but the EcmaXml only shows one (and the type is being updated in this latest update?)

File: xml/Microsoft.Win32.SafeHandles/SafeMemoryMappedFileHandle.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (joelmartinez) [reply xml/Microsoft.Win32.SafeHandles/SafeMemoryMappedFileHandle.xml]: offhand, no I'm not sure why this occurred ... yes a bug would be ideal

**Pull request merged**